tests/build-images-2019.ps1

```diff
@@ -3,7 +3,7 @@ Set-PSDebug -Trace 2
 # Exit on error.
 $ErrorActionPreference = "Stop"
 
-$ver = 'NATS_SERVER 2.9.24'.Split(' ')[1]
```

Comment:
Did we forget to bump them before?

Reply:
No, there are one set of tests to test the whichever version is being released.

Reply:
So they flip flop depending on which is being released since it doesn't happen at the same time (or shouldn't in the same PR for this repo).